src/tcia_utils/nbia.py

```diff
@@ -962,19 +980,21 @@ def manifestToList(manifest):
         first_line = f.readline()
         f.seek(0, 0)
         if "downloadServerUrl" in first_line:
-            print("Removing headers from TCIA mainfest.")
+            _log.info("Removing headers from TCIA mainfest.")
             # write lines to list
             for line in f:
                 data.append(line.rstrip())
             # remove the parameters from the list
             del data[:6]
-            print("Returning", len(data), "Series Instance UIDs (scans) as a list.")
+            _log.info(f"Returning {len(data)} Series Instance UIDs (scans) as a list.")
             return data
         else:
-            print("This is not a TCIA manifest file, or you've already removed the header lines.")
+            _log.error(
```

Comment:
I think this one can be a warning since the function will still return the series UIDs.  This step just prevents someone from deleting the first six lines in the file if there is no manifest header.

Reply:
Makes sense; the call has been changed to .warning()